tests/hpc/gnu_compilers_hpc.pm

```diff
@@ -12,23 +12,27 @@ use Mojo::Base qw(hpcbase), -signatures;
 use testapi;
 use serial_terminal 'select_serial_terminal';
 use utils;
+use version_utils;
 use Utils::Logging qw(export_logs);
 
 sub run ($self) {
     select_serial_terminal();
-    my $version = get_required_var('GNU_COMPILERS_HPC_VERSION');
+    my $version = get_var('GNU_COMPILERS_HPC_VERSION', '');
+    my $default_version = check_version('>=15-SP0', get_var('VERSION'), qr/\d{2}(?:-sp\d)?/) ? '7' : '4.8';
```

Comment:
Could you please comment above logic? Just make more obvious why its compared with 15-SP0, why we have here numbers 7 and 4.8.

Reply:
done.